study.md

```diff
@@ -87,14 +87,19 @@ Create an object literal named `vega` with the following properties and values.
 | solarMass | 2.135 |
 
 ```javascript
-// your answer here
+// let vega = {
+distance: 25.04,
+solarMass: 2.135,
+};
 ```
+<!-- I am still not 100% this is what you're asking for. If it is not can you give me the specific repo or document I should reference to review it? -->
 
 ## Ruby vs. JavaScript
 
 If, in the above code, `var`, `let`, or `const` were removed and the code were
 executed as Ruby code, what type of object will be created?
 
 ```md
-<!-- your answer here -->
+<!--If I am correct it would just declare a variable in JS, becuase it didn't use class and didn't capitolize the Vega. -->
```

Comment:
It will create a hash with the keys "distance" and "solarMass" as symbols.  Their values are 25.04 and 2.135, respectively.